ccm-connector/src/main/java/com/sequenceiq/cloudbreak/ccmimpl/altus/MinaSshdManagementClient.java

```diff
@@ -184,6 +237,50 @@ public GenerateAndRegisterSshTunnelingKeyPairResponse generateAndRegisterSshTunn
         }
     }
 
+    /**
+     * Wraps call to registerSshTunnelingKey.
+     *
+     * @param requestId         the request ID for the request
+     * @param accountId         the account ID
+     * @param minaSshdServiceId the minasshd service ID
+     * @param keyId             the key ID
+     * @param publicKey         the public key
+     * @throws CcmException if an exception occurs
+     */
+    public RegisterSshTunnelingKeyResponse registerSshTunnelingKey(String requestId, String accountId, String minaSshdServiceId,
+            String keyId, PublicKey publicKey) throws CcmException {
+        checkNotNull(requestId, "requestId should not be null.");
+        checkNotNull(accountId, "accountId should not be null.");
+        checkNotNull(minaSshdServiceId);
+        checkNotNull(keyId);
+        checkNotNull(publicKey);
+
+        MinaSshdManagementBlockingStub blockingStub = newStub(requestId);
+        RegisterSshTunnelingKeyRequest.Builder requestBuilder = RegisterSshTunnelingKeyRequest.newBuilder()
+                .setMinaSshdServiceId(minaSshdServiceId)
+                .setKeyId(keyId)
+                .setAccountId(accountId)
+                .setPublicKey(publicKey.getPublicKey())
+                .setPublicKeyAlgorithm(publicKey.getPublicKeyAlgorithm());
+
+        try {
+            LOGGER.debug("Calling registerSshTunnelingKey with requestId: {}, accountId: {}, minaSshdServiceId: {}",
+                    requestId, accountId, minaSshdServiceId);
+            RegisterSshTunnelingKeyResponse response = blockingStub.registerSshTunnelingKey(requestBuilder.build());
+            if (response == null) {
+                throw new CcmException("Got null response from MinaSshdManagementService registerSshTunnelingKey gRPC call", false);
+            }
+            return response;
+        } catch (StatusRuntimeException e) {
+            String message = "MinaSshdManagementService registerSshTunnelingKey gRPC call failed: " + e.getMessage();
+            Status status = e.getStatus();
+            Status.Code code = status.getCode();
+            boolean retryable = GrpcUtil.isRetryable(code);
+            LOGGER.debug("Got status code: {}, retryable: {}", code, retryable);
```

Comment:
Every log in this file uses `debug` which is why I did so here. I have hear conversations about using debug when possible so as to limit the amount of log clutter that we have, which I think is fair and understandable. I have `info` and `error` level logging around the calls to this method so I think this will be okay as is.